flutter/cpp/datasets/README.md

```diff
@@ -126,3 +126,68 @@ you need to replace `squad_eval_mini.tfrecord` by `squad_eval.tfrecord` in the
       --ground_truth_directory=/tmp/ade20k_512/annotations/raw  \
       --num_threads=4
     ```
+
+## SNU SR set
+
+preparing datasets and models
+
+1. dataset:
+
+    1. rotate 1080x1920 ones to 1920x1080 so that all the images are 1920x1080
+
+    2. convert all 1920x1080 jpg images to .png files
+
+    3. also generate scaled down 960x540 images
+
+2. model:
+
+   1. generate a modelswith 960x540 input and 1920x1080 output
+
+3. test with command line on Android device:
+
+   Build the main command line program and tflite backend
+
+```shell
+bazel build --config android_arm64 -c opt \
+  flutter/cpp/binary:main mobile_back_tflite:tflitebackend
+```
+
+  push them to the target Android device
+
+```shell
+adb push bazel-bin/flutter/cpp/binary/main /data/local/tmp/sr/main_sr
+adb push bazel-bin/mobile_back_tflite/cpp/backend_tflite/libtflitebackend.so \
+  /data/local/tmp/sr/
+```
+
+  Assuming we have the dataset and the model on the devices at
+  `/data/local/tmp/sr/dataset/` and `/data/local/tmp/edsr/tflite/pl_f32b5.tflite`
+
+```shell
+adb shell /data/local/tmp/sr/main_sr external snusr --mode=PerformanceOnly \
+  --output_dir=/data/local/tmp/sr_output \
+  --model_file=/data/local/tmp/edsr/tflite/pl_f32b5.tflite \
+  --images_directory=/data/local/tmp/sr/dataset/LR_jpg \
+  --ground_truth_directory=/data/local/tmp/sr/dataset/HR_jpg \
+  --lib_path=/data/local/tmp/sr/libtflitebackend.so
+```
+
+Or we can test on a host machine
+
+On a x86 machine running Ubuntu 22.04,
+
+```shell
+bazel build -c opt flutter/cpp/binary:main mobile_back_tflite:tflitebackend \
+  --host_cxxopt=-std=c++17 --cxxopt=-std=c++17 --copt=-march=native
```

Comment:
The guide uses `c++17` but the app uses `c++14`. Would it be an issue?

https://github.com/mlcommons/mobile_app_open/blob/769d0e4ffe1a8f9eb4e532a6f55ff9f5eb05e284/.bazelrc#L24-L31

Reply:
Should not be a problem. Newer TensorFlow uses C++17, so I just used C++17. I'll change it to C++14 for consistency. Thanks.